### ISSUE TITLE
`prefer-export-from`: Missed case

### ISSUE THREAD

**Issue by fisker** [sindresorhus/eslint-plugin-unicorn#2572]
### Description

https://github.com/sindresorhus/builtin-modules/blob/main/index.js#L1C1-L3C31

```js
import builtinModules from './builtin-modules.json' with {type: 'json'};

export default builtinModules;
```

Should write as

```js
export {default} from './builtin-modules.json' with {type: 'json'};
```

Should be caught by this rule. Not sure if this case is missed, but I'm sure it doesn't work well with import attributes, need investigate.
